build/yamls/base/conf/antrea-controller.conf

```diff
@@ -36,3 +36,10 @@ featureGates:
 
 # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
 #tlsMinVersion:
+
+# Enable mirroring legacy CRDs to new CRDs. If Antrea is upgraded from version <= v0.13 and
+# legacy CRDs is used, do enabled this feature, otherwise the CRDs created with legacy API
+# will not take any effect and work as expected. The legacy CRDs will be converted to new CRDs
```

Comment:
When the mirroring is enabled, the legacy CRDs will be...

Reply:
Thanks